byway/src/main/java/com/google/sps/servlets/StopsServlet.java

```diff
@@ -26,26 +37,49 @@
 /** Servlet that handles data for stops */
 @WebServlet("/api/stop")
 public final class StopsServlet extends HttpServlet {
-  @GuardedBy("this")
-  private ArrayList<String> stops = new ArrayList<String>();
+  private static final Type ARRAYLIST_STRING = new TypeToken<ArrayList<String>>() {}.getType();
+  private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+  private static final int SC_NO_CONTENT = 204;
+  private static final int SC_NOT_FOUND = 404;
```

Comment:
There are already constants for these declared in `HttpServletResponse`, just use those.
https://docs.oracle.com/javaee/7/api/javax/servlet/http/HttpServletResponse.html#SC_NOT_FOUND

Reply:
Right, I just saw the list of status codes and numbers and decided to go off that instead.  Changed.